src/app/RootStore.tsx

```diff
@@ -1,15 +1,18 @@
 import { NavigationStore } from '@features/home/NavigationList/model/navigation.store';
 import { UserStore } from '@entities/user/model/user.store';
 import { SidebarStore } from '@widgets/Sidebar/model/sidebar.store';
+import { ChartStore } from '@widgets/Chart/model/chart.store';
```

Comment:
unnecessary, we have expenses.store, add an array for the expenses that are sorted by category from backend